contracts/examples/digital-cash/src/digital_cash.rs

```diff
@@ -68,15 +78,35 @@ pub trait DigitalCash {
             deposit.expiration_round >= self.blockchain().get_block_round(),
             "deposit expired"
         );
+
+        const HASH_DATA_BUFFER_LEN: usize = 1024;
```

Comment:
Ed25519 signatures are 512 bits long, so this constant can be reduced (and also renamed)
```suggestion
        const SIGNATURE_BUFFER_LEN: usize = 64;
```